sickbeard/webserve.py

```diff
@@ -443,15 +443,6 @@ def setHistoryLayout(self, layout):
 
         return self.redirect("/history/")
 
-    def setHistoryLimit(self, history_limit):
```

Comment:
Except for the type check, I moved that part line 3584 (just below), no? Or did I forget something?